### PR TITLE
WIP Working version of CI / specify filename

### PR DESCRIPTION
**Pull request by szeck87**
This is still WIP.  Need to test out passing in the `--file-path` and find a better way to find global packages.

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -35,8 +59,7 @@ interface IArgs extends program.Command {
   filePath?: string;
 }
 
-const programArgs: IArgs = program.parse(process.argv);
```
Comment (Glavin001) [inline src/cli.ts]: Nope. `IArgs` should be used here.

File: src/index.ts [hunk 2]
```diff
@@ -57,3 +57,11 @@ export function loadInstalledBeautifiers(): Promise<Unibeautify> {
 export function setupUnibeautify(): Promise<Unibeautify> {
   return loadInstalledBeautifiers();
 }
+
+// TODO filter languages based on beautifiers that are installed
+export function getSupportedLanguages(): string[] {
```
Comment (Glavin001) [inline src/index.ts]: Isn't there a Unibeautify core method for this?

File: src/cli.ts (hunk 1 above)
Comment (szeck87) [reply src/cli.ts]: Whoops, removed that by accident.

File: src/index.ts (hunk 2 above)
Comment (szeck87) [reply src/index.ts]: Yea, https://github.com/Unibeautify/unibeautify/blob/master/src/beautifier.ts#L279-L283.  Was wanting to see how it looks outputting a lot of lines.

**Comment by Glavin001** [comment]
@szeck87 I made some changes and created a new PR to merge into this one: https://github.com/Unibeautify/unibeautify-cli/pull/26

**Comment by Glavin001** [comment]
@szeck87 Take a look at how https://github.com/h2non/requireg/blob/master/test/requiregSpec.js patches environments variables like `process.env.NODE_PATH` and maybe there is something we can do for our tests.

**Comment by szeck87** [comment]
> @szeck87 Take a look at how https://github.com/h2non/requireg/blob/master/test/requiregSpec.js patches environments variables like process.env.NODE_PATH and maybe there is something we can do for our tests.

What specifically?

**Pull request merged**